ABDataCollectionCore.js

```diff
@@ -3236,11 +3236,13 @@ module.exports = class ABDataCollectionCore extends ABMLClass {
     * @method eventRemove()
     * unsubscribe a event.
     *
+    * @param {object} emitter - Emitter object
+    * @param {string} eventName - Name of the event
     */
-   eventRemove(eventName) {
-      if (this.__events?.length > 0 && eventName) {
+   eventRemove(emitter, eventName) {
```

Comment:
Do we use this method any where else? Should we add the emitter as the second arg for compatibility?

If you think it's ok, can merge.

Reply:
Thank you.

I checked it is used only one place but it maybe used in Mobile code.

I updated it following your suggestion.